mixer/adapter/list/list.go

```diff
@@ -79,7 +80,28 @@ func (h *handler) HandleListEntry(_ context.Context, entry *listentry.Instance)
 		return adapter.CheckResult{}, err
 	}
 
-	found, err := l.checkList(entry.Value)
+	var value string
+	isString := true
+
+	switch entryVal := entry.Value.(type) {
+	case *v1beta1.Value:
+		if entryVal.GetStringValue() != "" {
+			value = entryVal.GetStringValue()
+		} else if entryVal.GetIpAddressValue() != nil {
+			value = string(entryVal.GetIpAddressValue().Value[:])
+			isString = false
+		} else {
+			return getCheckResult(h.config, rpc.INVALID_ARGUMENT, fmt.Sprintf("%v is not a valid string or IP address", entryVal)), nil
+		}
+	case string:
```

Comment:
Given the updated template, when does string and []byte ever happen?

Reply:
I think this should not be happening. It's worth tracking down why these values are coming in during that test, it probably means there's something incorrect in the test setup.

Reply:
We need both as oop adapter sends the v1beta1.value* objects and in process adapter sends plane byte and string interface